### ISSUE TITLE
Can Flowtron be used for small dataset(only a few minutes)?

### ISSUE THREAD

**Issue by blx0102** [NVIDIA/flowtron#45]
As I know, LibriTTS dataset has some speakers with only several minutes data. Although speaker with <5m data were filtered out as the paper mentioned, there should still contain some speakers with lacking data. 
So it has proved that small dataset can be studied well as far as it was trained with many other data together. But what if I only have a small dataset as well as a pre-trained LibriTTS model?
I tried to fine tune a 5-minutes dataset on the LibriTTS model, but can't get any reasonable result.
Is there any powerful method to fine tune or adapt those small datasets from a pre-trained model?

**Comment by rafaelvalle** [comment]
@blx0102 can you generate validation loss and attention maps on your 5-minute dataset ?

**Comment by blx0102** [comment]
@rafaelvalle Thanks for your reply!

Some details for my test:
1. Learning rate was set to 5e-5
2. Used the pre-trained "flowtron_libritts.pt" model as checkpoint path, and set ignore_layers = ["speaker_embedding.weight"]
3. p_arpabet was set to 0.0

As the plot shows, the model has obvious overfitting. And both attentions are getting worse.
![image](https://user-images.githubusercontent.com/17761044/88630446-31cea000-d0e3-11ea-963a-05a540fce722.png)
![image](https://user-images.githubusercontent.com/17761044/88630600-6c383d00-d0e3-11ea-9dc0-3642a01b1be1.png)
![image](https://user-images.githubusercontent.com/17761044/88630636-7b1eef80-d0e3-11ea-91d8-2634b633d9c6.png)
![image](https://user-images.githubusercontent.com/17761044/88630651-8245fd80-d0e3-11ea-869f-48132649f494.png)

**Comment by rafaelvalle** [comment]
Can you do try this with `p_arpabet=1.0`?

**Comment by jimzers** [comment]
Hey @blx0102 , was wondering how you set up your 5 minute dataset. Would love to hear your answer!
